vercel/turbo
# use vc token if logging into Vercel

### Description
If users are logging into Vercel, we should allow for them to use an existing Vercel CLI token if it exists. This is only for non-SSO logins. SSO will follow in another PR.

### Testing Instructions
1. Use `vc login` to get a Vercel token. Do _not_ use SSO (this doesn't implement that yet).
2. Make sure you don't have a `turbo` token by running `turbo logout`.
3. Run `turbo login` - you should see a message saying `Existing Vercel token found!`
4. Run `turbo link` on a repo you have access to in Vercel
5. Try running a cacheable task. You should see that remote caching is enabled.

Closes TURBO-2194
